### ISSUE TITLE
add logical pages support

### ISSUE THREAD

**Issue by ale03068** [janishar/mit-deep-learning-book-pdf#23]
Hi, it's possible to add the support for numbering the pages like "VII (7 of 800)" when in the preamble and then like "20 (27 of 800)? In this way it's easier to reference the content of a specific page. (e.g "go to the example at page n")

**Comment by Tanmaysukhdeve** [comment]
Yes, it's possible to add page numbering as you described, with both Roman numerals for the preamble and Arabic numerals for the main content of a document. This is a common formatting style for many academic and formal documents. Here's how you can achieve this:

1. Section Breaks: Divide your document into two sections: one for the preamble and another for the main content. Most word processing software, like Microsoft Word or LaTeX, allows you to insert section breaks.

2. Page Numbering Format:
   - For the preamble section, set the page numbering format to use Roman numerals. This can usually be done in the header/footer settings of your document editor.
   - For the main content section, set the page numbering format to use Arabic numerals.

3. Page Numbering Display:
    In the preamble section, insert a field or placeholder in the header/footer where you want the Roman numeral page number to appear (e.g., "VII").
    In the main content section, insert another field or placeholder for the Arabic numeral page number.

4. Total Page Count:
    To display the total page count (e.g., "7 of 800" or "27 of 800"), insert a field or formula that calculates the total number of pages in the document and displays it in the appropriate format. This will vary depending on your document editor.

Here's a more detailed example for Microsoft Word:

For the Preamble (Roman numerals):

1. Go to the section of your document that contains the preamble.
2. Insert a section break to separate it from the main content.
3. Double-click in the header or footer area of the preamble section.
4. Insert a page number field by going to "Insert" > "Page Number" > "Format Page Numbers."
5. Choose "Roman numerals" for the numbering format.

For the Main Content (Arabic numerals):

1. Go to the section of your document that contains the main content.
2. Double-click in the header or footer area of the main content section.
3. Insert a page number field.
4. To display the total page count as "27 of 800," insert a field for the current page number and a field for the total number of pages in the document, then format them accordingly (e.g., "Page {PAGE} of {NUMPAGES}").

Please note that the exact steps may vary depending on the software you are using, so consult the documentation for your specific word processing or document layout tool for precise instructions.
